pkg/chartutil/requirements.go

```diff
@@ -258,6 +258,24 @@ func ProcessRequirementsEnabled(c *chart.Chart, v *chart.Config) error {
 	}
 
 	var chartDependencies []*chart.Chart
+	// If any dependency is not a part of requirements.yaml
+	// then this should be added to chartDependencies.
+	// However, if the dependency is already specified in requirements.yaml
+	// we should not add it, as it would be anyways processed from requirements.yaml
+
```

Comment:
I kinda prefer the more verbose description, since I have the long-term memory of a sea slug. :-) But I'm okay with whatever.